### PR TITLE
ISPN-6225 Allow the ratio segments/shard to be configurable for the A…

### PR DESCRIPTION
**Pull request by gustavonalle**
…ffinityIndexManager

https://issues.jboss.org/browse/ISPN-6225

Cannot reopen the previous PR https://github.com/infinispan/infinispan/pull/4576 due to https://github.com/isaacs/github/issues/361, so creating a new one

### CODE REVIEW

File: query/pom.xml [hunk 1]
```diff
@@ -145,19 +145,20 @@
          <artifactId>mockito-all</artifactId>
          <scope>test</scope>
       </dependency>
-
+      <dependency>
+         <groupId>org.latencyutils</groupId>
+         <artifactId>LatencyUtils</artifactId>
```
Comment (tristantarrant) [inline query/pom.xml]: Should be <scope>test</scope>

**Comment by gustavonalle** [comment]
@tristantarrant fixed

**Comment by tristantarrant** [comment]
This looks fine. However it is missing some documentation as to why and how one would configure a different ratio

**Comment by gustavonalle** [comment]
@tristantarrant I will include the doc in another PR, which will describe in detail all index manager available: the InfinispanIndexManager, ElasticsearchIndexManager, AffinityIndexManager and DirectoryBasedIndexManager

**Comment by tristantarrant** [comment]
I'll have to trust you on that :)
Merged !

**Pull request merged**